pkg/volume/glusterfs/glusterfs.go

```diff
@@ -255,8 +260,19 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 	for hostIP := range addr {
 		errs = b.mounter.Mount(hostIP+":"+b.path, dir, "glusterfs", options)
 		if errs == nil {
+			glog.Infof("glusterfs: successfully mounted %s", dir)
 			return nil
 		}
 	}
+
+	// Failed mount scenario.
+	// Since gluster does not return eror text
+	// it all goes in a log file, we will read the log file
+	logerror := readGlusterLog(log, b.pod.Name)
+	if logerror != nil {
+		glog.Errorf("glusterfs: mount failed: %v", logerror)
+		return fmt.Errorf("glusterfs: mount failed: %v", logerror)
```

Comment:
@rootfs - I'm not sure we want to return errs as well in this case ... I say this only because, errs only comes from the mounter
 ```errs = b.mounter.Mount(hostIP+":"+b.path, dir, "glusterfs", options) ```

and since this is gluster, the useful error information is in the log, if we also return *errs* we will get redundancy in the fact that we are asking the users to look at the log that we are already exposing and a useless exit status code error (which is pretty standard and as stated, useless)  ... see below

```
  35s		35s		1	{kubelet 127.0.0.1}			Warning		FailedMount	Unable to mount volumes for pod "bb-multi-pod1_default(0e0022ec-0ca4-11e6-83ff-52540092b5fb)": glusterfs: mount failed: 
[2016-04-27 18:19:57.717843] E [socket.c:2332:socket_connect_finish] 0-glusterfs: connection to 192.168.123.222:24007 failed (Connection timed out)
[2016-04-27 18:19:57.718122] E [glusterfsd-mgmt.c:1819:mgmt_rpc_notify] 0-glusterfsd-mgmt: failed to connect with remote-host: 192.168.123.222 (Transport endpoint is not connected)
 
Mount failed: exit status 1
Mounting arguments: 192.168.123.222:myVol2 /var/lib/kubelet/pods/0e0022ec-0ca4-11e6-83ff-52540092b5fb/volumes/kubernetes.io~glusterfs/pv-gluster2 glusterfs [log-level=ERROR log-file=/var/lib/kubelet/plugins/kubernetes.io/glusterfs/pv-gluster2/bb-multi-pod1-glusterfs.log]
Output: Mount failed. Please check the log file for more details.

```

What do you think?